### PR TITLE
fixed: Update typography modal  and add delete button to shadow component #396

### PR DESCRIPTION
**Pull request by noobCode-69**
Fixed the issue Update typography modal #396
* In the ThemeProvider.ts provided a theme for Input and Select
* Changed the order of button
* Change the save button color from green to grey

### CODE REVIEW

**Review by Pranav2612000 — APPROVED** [review]

**Comment by noobCode-69** [comment]
Please Merge and Close

**Comment by alexdanilowicz** [comment]
@noobCode-69, thanks for the PR. Are you in our Slack channel? https://join.slack.com/t/mirrorful/shared_invite/zt-1ps2xtxh0-2NaixFfFzSKZbr5gw_AHfA

**Comment by alexdanilowicz** [comment]
Hey @noobCode-69 could we clean up the hover here? I think the background should be nothing and instead the font just gets bolder. 
Screen Shot 2023-05-08 at 9 51 01 PM

File: packages/core/src/components/Shadows/EditShadowModal.tsx [hunk 1]
```diff
@@ -335,10 +362,14 @@ export function EditShadowModal({
                       color: index === shadowIndex ? 'black' : 'gray',
                       width: '65px',
                       margin: '10px',
+                      position: 'relative',
                     }}
                     onClick={() => setShadowIndex(index)}
                   >
                     {index + 1}
+                    <Box position="absolute" top="-5px" right="-5px">
+                      <CloseButton size="sm" onClick={handleDelete(index)} />
```
Comment (alexdanilowicz) [inline packages/core/src/components/Shadows/EditShadowModal.tsx:371]: let's remove the hover styling and instead add our own _hover styling to make the icon bolder 

also, what does size xs look like?

File: packages/core/src/components/Typography/FontSize/EditFontSizeModal.tsx [hunk 2]
```diff
@@ -186,6 +179,9 @@ export function EditFontSizeModal({
                 />
               </>
             )}
+            <Button onClick={handleSave} css={{ marginLeft: '12px' }}>
```
Comment (alexdanilowicz) [inline packages/core/src/components/Typography/FontSize/EditFontSizeModal.tsx:182]: Great work on the typography stuff!  🚀

File: packages/core/src/components/Shadows/EditShadowModal.tsx (hunk 1 above)
Comment (noobCode-69) [reply packages/core/src/components/Shadows/EditShadowModal.tsx:371]: * default (no hover state) => no background color && normal icon boldness
* on hover =>  no background color && icon gets bolder
Is this the desired behaviour

Comment (alexdanilowicz) [reply packages/core/src/components/Shadows/EditShadowModal.tsx:371]: yes! @noobCode-69

**Comment by alexdanilowicz** [comment]
@noobCode-69 if you're able to make the requested changes here, happy to merge it in

**Comment by noobCode-69** [comment]
> @noobCode-69 if you're able to make the requested changes here, happy to merge it in

Yeah will be it tomorrow

**Comment by noobCode-69** [comment]
Hey @alexdanilowicz made all the changes mentioned here
* default (no hover state) => no background color && normal icon boldness
* on hover => no background color && icon gets bolder
Just merge it,
from now on will make seperate PR

**Comment by noobCode-69** [comment]
Hey @alexdanilowicz  merge this!!

**Comment by alexdanilowicz** [comment]
@noobCode-69 ok, taking a look right now.

**Pull request merged**